### ISSUE TITLE
🐛 BUG: SVG files are not loaded when using `wrangler pages dev ./dist`

### ISSUE THREAD

**Issue by CMCDragonkai** [cloudflare/wrangler2#1561]
### What version of `Wrangler` are you using?

2.0.23

### What operating system are you using?

Linux

### Describe the Bug

I used `wrangler pages dev ./dist` on https://github.com/MatrixAI/Polykey-Docs/, and it didn't resolve the `/images/Polykey Logo.svg` file, even though this exists.

It does work on the published site. Seems like the wrangler doesn't realise that svgs are also valid images/files.

It shows the 404 page instead.

**Comment by Skye-31** [comment]
Hi, I can't repro this on wrangler 2.0.24. Are you still experiencing this? 
If so could you provide an up to date version of your repo as the instructions for that seem to be out of date.

**Comment by CMCDragonkai** [comment]
I stopped using cloudflare pages, and have transitioned to using cloudflare workers instead. I haven't actually tested with svg files yet. I have a feeling this is not due to svg file necessarily but actually due to spaces in the file names: https://community.cloudflare.com/t/serving-files-images-with-spaces-in-their-filenames-from-cf-workers-sites/356617

I worked around this by by removing spaces and replacing with `-` character.
